homeassistant/components/tuya/sensor.py

```diff
@@ -818,6 +825,35 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             native_unit_of_measurement=UnitOfElectricPotential.VOLT,
             subkey="voltage",
         ),
+        TuyaSensorEntityDescription(
+            key=DPCode.ADD_ELE,
+            name="add ele",
```

Comment:
You can't add names. You need to set up translations.

Reply:
fixed in the new commits.